Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:14
+
+LABEL maintainer="Ryan Blunden <ryan.blunden@ext.doppler.com>"
+
+# DOPPLER_TOKEN must be supplied as a build-arg
+ARG DOPPLER_TOKEN
+ENV DOPPLER_TOKEN ${DOPPLER_TOKEN}
+
+# Install the Doppler CLI
+RUN curl -Ls https://cli.doppler.com/install.sh | sh
+
+# Execute `doppler run` so it creates a secrets fallback file
+RUN doppler run -- echo "Create secrets fallback file"
+
+WORKDIR /usr/src/app
+COPY ["package.json", "package-lock.json*", "src", "./"]
```

Comment:
Are the first three args redundant here? I believe the last arg copies everything, but I may be mistaken.

Reply:
The last arg specifies the path to where the files should be copied. Also, I shouldn't have copied "src" here, as that defeats the purpose of only invalidating the cache if the deps have changed.

Reply:
Duh, I forgot that standard `cp` semantics apply here.